Tweetinvi.Core/Public/Models/Location.cs

```diff
@@ -22,22 +22,18 @@ public static bool CoordinatesLocatedIn(ICoordinates coordinates, ILocation loca
             return CoordinatesLocatedIn(coordinates, location.Coordinate1, location.Coordinate2);
         }
 
-        public static bool CoordinatesLocatedIn(ICoordinates coordinates, ICoordinates boxCoordinates1, ICoordinates boxCoordinates2)
-        {
-            bool xIsBelowCoord1AndAboveCoord2 = boxCoordinates1.Longitude >= coordinates.Longitude &&
-                                                    coordinates.Longitude >= boxCoordinates2.Longitude;
+		public static bool CoordinatesLocatedIn(ICoordinates testPoint, ICoordinates boxNorthwestCoordinates, ICoordinates boxSoutheastCoordinates)
```

Comment:
I do not think this is correct because we do not know the order of the coordinates that we receive. The first coordinate can be the south east, the north east or anything else.